src/lib/monaco-editor/json.worker.ts

```diff
@@ -0,0 +1,209 @@
+/**
+ * Source: https://github.com/microsoft/monaco-editor/blob/35eb0efbc039827432002ccc17b120eb0874d70f/src/language/json/jsonWorker.ts
```

Comment:
Do you foresee this needing to be synchronised from time to time?

Reply:
I think the `JSONWorker` class is pretty "stable", e.g. looking at the [file history](https://github.com/microsoft/monaco-editor/commits/main/src/language/json/jsonWorker.ts) there aren't frequent updates. So I don't foresee a need to sync this often.

Side note: I tried to use the native JS feature to extend a class (because that would greatly reduce the copy+pasted code), but unfortunately `JSONWorker` isn't exported as a JS module in the transpiled/packaged code of `monaco-editor`.